src/rocksdb_options.rs

```diff
@@ -77,8 +77,13 @@ impl BlockBasedOptions {
         }
     }
 
-    pub fn set_lru_cache(&mut self, size: size_t) {
-        let cache = crocksdb_ffi::new_cache(size);
+    // the recommanded num_shard_bits should be calculated like this:
+    //     num_shard_bits = (size + 4194303) / 4194304 (4MB)
+    // for example, if your size is 4GB, num_shard_bits = (4 * 1024^3 + 4194303) / 4194304 = 10
+    // the recommanded strict_capacity_limit is false if your memory is sufficient
+    // the recommanded high_pri_pool_ratio should be 0.05 or 0.1
+    pub fn set_lru_cache(&mut self, size: size_t, num_shard_bits: c_int, strict_capacity_limit: c_uchar, high_pri_pool_ratio: c_double) {
```

Comment:
s/c_uchar/bool
s/c_double/f64

Reply:
it won't pass compiling because c does not support bool type, it will cause an error when rust calls c functions.